src/devices/Seesaw/SeesawGpio.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.Seesaw
+{
+    public partial class Seesaw : IDisposable
+    {
+
+        /// <summary>
+        /// Set the PinMode for a GPIO Pin. 
+        /// </summary>
+        /// <param name="pin">The pin that has its mode set.</param>
+        /// <param name="mode">The pin mode to be set.</param>
+        public void SetGpioPinMode(byte pin, PinMode mode)
+        {
+            if (pin < 0 || pin > 63)
+            {
+                throw new ArgumentException("Gpio pin must be within 0-63 range.");
+            }
+
+            SetGpioPinModeBulk((UInt64)(1 << pin), mode);
+        }
+
+        /// <summary>
+        /// Set the PinMode for a number of GPIO pins
+        /// </summary>
+        /// <param name="pins">A 64bit integer containing 1 bit for each pin. If a bit is set to 1 then the pin mode is set for the associated pin.</param>
+        /// <param name="mode">The pin mode to be set.</param>
+        public void SetGpioPinModeBulk(UInt64 pins, PinMode mode)
+        {
+            if (!HasModule(Seesaw_Module.Gpio))
+            {
+                throw new NotSupportedException($"The hardware on I2C Bus {I2cDevice.ConnectionSettings.BusId}, Address 0x{I2cDevice.ConnectionSettings.DeviceAddress:X2} does not support Adafruit SeeSaw GPIO functionality");
+            }
+
+            byte[] buffer = new byte[] { (byte)(pins >> 24), (byte)(pins >> 16), (byte)(pins >> 8), (byte)pins, (byte)(pins >> 56), (byte)(pins >> 48), (byte)(pins >> 40), (byte)(pins >> 32)};
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.buffers.binary.binaryprimitives?view=netstandard-2.1

Reply:
@krwq Unfortunately the byte array is not formatted as a ulong converted ty bytes but as two uints representing the low and high 32 bits and so there is no obvious bit conversion that I can see.
I have created functions for this conversion and documented the byte array layout and it will be available in the next update

Reply:
Done in the latest PR update.